pkg/controller/certificates/issuing/issuing_controller.go

```diff
@@ -332,12 +333,19 @@ func (c *controller) ProcessItem(ctx context.Context, key string) error {
 }
 
 // failIssueCertificate will mark the Issuing condition of this Certificate as
-// failed, and log an appropriate event. The reason and message of the
-// condition will be that of the CertificateRequest condition passed.
+// false, set the Certificate's last failure time and issuance attempts and log
+// an appropriate event. The reason and message of the Issuing condition will be that of
```

Comment:
This feels like a violation of Oxford commas and friends.

You could do
> set the Certificate's last failure time and issuance attempts fields, and ...

The `fields` part enables one to get out of the box of too many `and`s in the sentence.